Kernel_23/include/CGAL/rational_rotation.h

```diff
@@ -78,8 +74,8 @@ rational_rotation_approximation( const NT &  dirx,     // dir.x()
       {
           p = p0 + p1;
           q = q0 + q1;
-          sin = NT(2)*p*q;
-          den = p*p + q*q;
+          sin = NT(2)*square(p);
+          den = square(p) + square(q);
```

Comment:
Might as well store `square(p)` to avoid computing it twice? Others also seem to be computed a few times.